efficientdet/backbone/efficientnet_model.py

```diff
@@ -248,10 +260,27 @@ def _build(self):
         padding='same',
         data_format=self._data_format,
         use_bias=False)
+
+
+    if self._block_args.fused_conv:
+      bn0_name = "tpu_batch_normalization"
```

Comment:
Do you need these changes? Since they are already keras layers and it works well with existing code?

Reply:
Yes, I need them, because without them variable names are different. 
When keras model created in the context of `tf.Graph().as_default` you can go without this naming and variables names will be equal with tensorflow 1 model version, but when keras model created without this context, variables names are different. I changed the `test_model_output_variables` test to handle it.

Reply:
You shouldn't remove graph test case.